kernel/kernel-api/src/main/java/io/delta/kernel/internal/TransactionImpl.java

```diff
@@ -457,8 +462,10 @@ private TransactionCommitResult doCommit(
                           // TODO: handle RemoveFiles.
                           if (!action.isNullAt(ADD_FILE_ORDINAL)) {
```

Comment:
can you add if this action is a remove, we throw exception, that way we get to update the file size histogram when remove support is added.